Protos/V1/partitions_common.proto

```diff
@@ -115,3 +117,82 @@ message GetPartitionRequest {
 message GetPartitionResponse {
   PartitionRaw partition = 1; /** The raw partition. */
 }
+
+/**
+ * Request to update multiple partitions.
+ *
+ * Cannot update more than 30 partitions
+ */
+message UpdatePartitionsRequest {
+  /**
+   * Represents the data of the partitions that will be updated.
+   *
+   * ID is mandatory.
+   * -1 means that value will not be updated
+   * value >= 0 means that it will be the new value
+   */
+  message PartitionUpdate {
+    string id = 1; /** The ID of the partition that will be updated. */
+    int64 pod_reserved = 2; /** Whether the partition is reserved for pods. */
+    int64 pod_max = 3; /** The maximum number of pods that can be used by sessions using the partition. */
+    int64 preemption_percentage = 4; /** The percentage of the partition that can be preempted. */
+    int64 priority = 5; /** The priority of the partition. */
+  }
+
+  repeated PartitionUpdate partitions = 1; /** The list of partitions to update. */
+}
+
+/**
+ * Response to update a partition.
+ *
+ * Return a list of raw partitions with the new values.
+ */
+message UpdatePartitionsResponse {
+  repeated PartitionRaw partitions = 1; /** The list of raw partitions. */
+}
+
+/**
+ * Request to delete partitions.
+ *
+ * If the ID is not found, it is ignored
+ */
+message DeletePartitionsRequest {
+  repeated string id = 1; /** The IDs of the partitions to delete. */
+}
+
+/**
+ * Response to delete partitions.
+ */
+message DeletePartitionsResponse {
+  Empty ok = 1;
+}
+
+/**
+ * Request to create multiple partitions.
+ *
+ * Cannot create more than 30 partitions
+ */
+message CreatePartitionsRequest {
+  /**
+   * Represents the data of the partitions that will be created.
+   */
+  message PartitionCreate {
+    repeated string parent_partition_ids = 2; /** The parent partition IDs. */
+    int64 pod_reserved = 3; /** Whether the partition is reserved for pods. */
+    int64 pod_max = 4; /** The maximum number of pods that can be used by sessions using the partition. */
+    map<string, string> pod_configuration = 7; /** The pod configuration. */
```

Comment:
As far as I remember, `map` is not well supported. I think we used a list of pairs in other places.